lib/vertica/protocol/frontend/startup.rb

```diff
@@ -8,12 +8,22 @@ def initialize(user, database, options = nil)
         @user     = user
         @database = database
         @options  = options
+        @type     = "vertica-rb"
+        @pid      = Process.pid.to_s
+        @platform = Gem::Platform.local.to_s
+        @version  = Vertica::VERSION
+        @label    = @type+"-"+@version+"-"+SecureRandom.hex(10) 
```

Comment:
Do you think it's worth it to have a regex to test the uuid validity? Or is the assert_match without the uuid enough?